setup.py

```diff
@@ -22,18 +22,22 @@
 
 name = 'cirq'
 
-# If CIRQ_DEV_VERSION is set then we use cirq-dev as the name of the package
-# and update the version to this value.
-if 'CIRQ_DEV_VERSION' in os.environ:
-    name = 'cirq-dev'
-    __version__ = os.environ['CIRQ_DEV_VERSION']
-
 description = ('A framework for creating, editing, and invoking '
                'Noisy Intermediate Scale Quantum (NISQ) circuits.')
 
 # README file as long_description.
 long_description = io.open('README.rst', encoding='utf-8').read()
 
+# If CIRQ_DEV_VERSION is set then we use cirq-dev as the name of the package
+# and update the version to this value.
+if 'CIRQ_DEV_VERSION' in os.environ:
+    name = 'cirq-dev'
+    __version__ = os.environ['CIRQ_DEV_VERSION']
+    long_description = (
+        "**This is a development version of Cirq and may be "
+        "unstable.**\n\n**For the latest stable release of Cirq "
+        "see ** `here <https://pypi.org/project/cirq>`__.\n" + long_description)
```

Comment:
I'm not familiar with reStructuredText but should this be a single underscore?
```suggestion
        "see ** `here <https://pypi.org/project/cirq>`_.\n\n" + long_description)
```

Reply:
Double underscore makes it an anonymous hyperlink, which is what we want.  With the single underscore you can reference it.  https://stackoverflow.com/questions/27420317/restructured-text-rst-http-link